coderedcms/models/page_models.py

```diff
@@ -1293,7 +1291,11 @@ def send_summary_mail(self, request, form, processed_data):
             message_args['reply_to'] = template_reply_to.render(context).split(',')
 
         # Send email
+        self.__send_mail(request, message_args)
+
+    def __send_mail(self, resuest, message_args):
```

Comment:
Let's make this public, e.g. `send_mail()` would be fine.

Reply:
Also, it doesn't look like the `request` arg is used/needed here.

Reply:
My use case has the next code, so request can not be removed
```python
mail_settings = MailSettings.for_request(request)
connection = get_connection(
    username=mail_settings.username, password=mail_settings.password, fail_silently=False
)
```